### DISCUSSION TITLE
PostgREST API - Relation does not exist error

### DISCUSSION THREAD

**Discussion by mchatrvd** [PostgREST/postgrest#3097] [Q&A] [answered]
I have a PostgREST service which appends a default schema to the table name that i pass. The table name passed already has another schema name appended to a table in it where the table resides, and which I will like to call through API. 
So if <Table A> resides in <Schema A> and I am passing <Schema A>.<Table A> as the table name, whereas API is appending default schema and is passing <Default Schema>.<Schema A>.<Table A> as the table name.
How can we fix for the API not to append the default schema name? Thanks in advance

**Comment by laurenceisla** [comment]
Hi. Sorry, I didn't catch most of your explanation... but I think you want to change the default `schema` that is used? That info can be found in the [Schemas section](https://postgrest.org/en/v12.0/references/api/schemas.html) in the docs.

If that's not it, please clarify what's the behavior you're getting vs the result you want, preferably with an example.

**Reply by mchatrvd** [reply]
Sorry for the confusion. So, the config file has the schema mentioned under parameter db-schemas as [Schema A]. So, when we fire a table name in the curl command, it appends the Schema name by default and does the GET operation. However, we want it Not to append the Schema Name, as we  will like user to mention table name along with the Schema Name to extract the data from the PostgreSQL table. So, some user may pass table name under Schema A in the form as Schema A.Table A and some other user may fire the same operation against another table in Schema B and may use Schema B.Table B

**Reply by laurenceisla** [reply, marked as answer]
Ah, I get it. The only way to change the schema is by using the `Accept-Profile: schema_name` (for `GET`, `HEAD`) and `Content-Profile: schema_name` (for `POST`, etc.) as it's mentioned in the docs I linked. For that to work, your configuration should list all the available schemas, e.g. `db-schema = schema1,schema2`
